static/js/egi.js

```diff
@@ -65,6 +65,7 @@ jQuery(document).ready(function () {
           $('.progress').stop(true, false, true).fadeToggle(100);
       });
   }
+};
```

Comment:
```suggestion
});
```
Apparently there is also a missing `)`.
That said I'm not sure what's the exact purpose of those changes, if they are still required, and what may not be correctly working if the current version is broken.

@thebe14 I think you are the one that added this, any thoughts?

Reply:
There is definitely missing `});` although I am not clear how I could commit that and still have the things work.